modules/QnA/resources/qnaService.js

```diff
@@ -164,7 +164,10 @@
             return this.cuePoint().text;
         };
 
-        this.isAnswerOnAirActive = function(playerTime) {
+        this.isAnswerOnAirActive = function(playerTime,player) {
+            if(player && player.embedPlayer.isDVR()){
```

Comment:
this looks like a bug player.embedPlayer.currentTime contains the absolute time, not relative one